ngi_reports/common/ign_sample_report.py

```diff
@@ -62,11 +60,6 @@ def __init__(self, config, LOG, working_dir, **kwargs):
         self.parse_picard_metrics()
         self.parse_deliveries()
 
-        self.create_aggregate_statistics()
-
-        self.LOG.debug('Plotting graphs')
-        self.make_plots()
```

Comment:
Yup, fine by me as long as there's a comment to avoid future me getting confused :) Fine with the node-specific calls.